Add test case for controller rendering with block (#1010)
github/view_component: docs/CHANGELOG.md

## main

> * Add test case for rendering from the controller with a block

Reviewer: ```suggestion
* Add test case for rendering a ViewComponent with slots in a controller.
```
